### DISCUSSION TITLE
Automated Code Documentation for AIR Apps? ie a "doxygen" for my huge flex app? ChatGTP?

### DISCUSSION THREAD

**Discussion by TaS-Sensys** [airsdk/Adobe-Runtime-Support#2511] [General]
Still navigating and adding features to an old 100K plus line AIR/Flex application - was supposed to be sun setting it, well...

The number of files and lines of code are boggling, after a year of poking at parts I really need a 10,000 ft view of how its put
together to see what is useful vs dead. Even just understanding the origins/ages of relative pieces would be helpful, as the
style and applied libraries evolved in the codebase. I need a map!

I use Intellij IDEA and an ancient flex 4.1.0.16076 SDK (newer versions = more unstable), with build processes completely migrated out of Adobe Flashbuilder (good riddance).

BTW I toyed with ChatGTP to assist with some flex/as3 coding - not fully trustworthy but might provide a way to migrate someday...

**Comment by radubirsan** [comment]
I am in the same situation as @TaS-Sensys except I did not do the migration to IDEA yet.
